resource-managers/kubernetes/core/src/test/scala/org/apache/spark/scheduler/cluster/k8s/ExecutorPodsAllocatorSuite.scala

```diff
@@ -58,6 +58,7 @@ class ExecutorPodsAllocatorSuite extends SparkFunSuite with BeforeAndAfter {
   private val conf = new SparkConf()
     .set(KUBERNETES_DRIVER_POD_NAME, driverPodName)
     .set(DYN_ALLOCATION_EXECUTOR_IDLE_TIMEOUT.key, "10s")
+    .set(KUBERNETES_DRIVER_SUBMIT_CHECK, true)
```

Comment:
Do we need this? The newly added test case `SPARK-35828` has it already.

Reply:
This whole test suite is just mock. Here we just set this to true to tell all the existing tests that they are still running in cluster mode. So we do need to modify them.